pkg/apis/app/v2/kieapp_types.go

```diff
@@ -252,6 +252,7 @@ type KieAppObject struct {
 	ImageTag string `json:"imageTag,omitempty"`
 	// The storageClassName to use
 	StorageClassName string `json:"storageClassName,omitempty"`
+	JBPMCluster      bool   `json:"jbpmCluster,omitempty"`
```

Comment:
Please add a description above this field entry and re-create the CRD

Reply:
Done

Reply:
Thanks